server/src/main/java/org/spine3/server/event/DispatcherEventDelivery.java

```diff
@@ -78,8 +81,10 @@ public static DispatcherEventDelivery directDelivery() {
     private static final class PredefinedDeliveryStrategies {
 
         /**
-         * A pre-defined instance of the {@code DispatcherEventDelivery}, which does not postpone any event dispatching
-         * and uses {@link com.google.common.util.concurrent.MoreExecutors#directExecutor()} for operation.
+         * A pre-defined instance of the {@code DispatcherEventDelivery}, which does not
+         * postpone any event dispatching and uses
+         * {@link com.google.common.util.concurrent.MoreExecutors#directExecutor() MoreExecutors.directExecutor()}
```

Comment:
The same:
`A pre-defined instance of the {@code DispatcherEventDelivery},`
`which does not postpone any event dispatching and uses`